src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -15,6 +15,9 @@
   <!-- We can only analyze the OpenAPI spec after the project has been built -->
   <Target Name="ValidateOpenApi" DependsOnTargets="ResolveProjectReferences" AfterTargets="Build">
     <PropertyGroup>
+      <!-- The path of the ASP.NET Core project being built -->
+      <StartupAssemblyPath Condition="'$(StartupAssemblyPath)' == ''">$(MSBuildProjectDirectory)\$(OutputPath)</StartupAssemblyPath>
```

Comment:
We could update the internal documentation reference: https://gsoftdev.atlassian.net/wiki/spaces/TEC/pages/3839557876/MSBuild+Parameters